src/joysensortype.h

```diff
@@ -0,0 +1,29 @@
+/* antimicrox Gamepad to KB+M event mapper
+ * Copyright (C) 2022 Max Maisel <max.maisel@posteo.de>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <QMetaType>
+
+enum JoySensorType
+{
+    ACCELEROMETER,
+    GYROSCOPE,
+    SENSOR_COUNT
+};
```

Comment:
I don't think this is worth it since in near future all SDL versions support sensors and this optimization will have no effect but clutters the header with an #ifdef.

Reply:
> in near future

This future is not as near as we would wish to. For example Ubuntu 20.04 which doesn't support (and won't) newer sdl is supported until 2025.